pkg/utils/utils.go

```diff
@@ -13,11 +14,15 @@ import (
 	"github.com/alecthomas/units"
 	v2vv1alpha1 "github.com/kubevirt/vm-import-operator/pkg/apis/v2v/v1alpha1"
 	k8svalidation "k8s.io/apimachinery/pkg/util/validation"
+	rclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 const (
 	// MaxLabelValueLength specifies max length of k8s label value
 	MaxLabelValueLength = 63
+
+	// RestoreVMFinalizer defines restore source vm finalizer
+	RestoreVMFinalizer = "vmimport.v2v.kubevirt.io/restore"
```

Comment:
RestoreVM has a different semantic in virt domain, perhaps rename it to RestoreVMStateFinalizer and the annotation name to "vmimport.v2v.kubevirt.io/restore-state"

Reply:
:+1: